scripts/src/main/resources/scripts/command/gh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+GH_HOME="${DEVON_IDE_HOME}/software/gh"
+
+function doSetup(){
+  # Install GitHub CLI in the software folder
+
+  if [ ! -d "${GH_HOME}" ]
```

Comment:
add `|| [ "${1}" != "silent" ]` and remove it from the next if statement, because you should return the version otherwise. Add this codesnippet after this if statement as alternative like
```
 if [ "${1}" != "silent" ] && ! doIsQuiet
  then
    doRunCommand "helm version"
  fi